stdlib/public/Concurrency/DiscardingTaskGroup.swift

```diff
@@ -158,6 +160,13 @@ public struct DiscardingTaskGroup {
     let _: Void? = try await _taskGroupWaitAll(group: _group, bodyError: nil)
   }
 
+  /// Adds a child task to the group.
+  ///
+  /// - Parameters:
+  ///   - priority: The priority of the operation task.
+  ///     Omit this parameter or pass `.unspecified`
+  ///     to set the child task's priority to the priority of the group.
+  ///   - operation: The operation to execute as part of the task group.
   @_alwaysEmitIntoClient
```

Comment:
missed to copy docs on this occurrence of the method

Reply:
I think it was intentional that we didn't copy/paste the documentation onto deprecated versions.